packages/components/collapsible-panel/src/collapsible-panel.styles.js

```diff
@@ -56,7 +56,7 @@ const getHeaderContainerStyles = (props, isOpen) => {
 
 const baseContainerStyles = css`
   position: relative;
-  min-width: 550px;
+  min-width: 284px;
```

Comment:
I suppose this is intended to be the value of the smallest supported constraint

```suggestion
  min-width: ${vars.constraint6};
```